helpers/auth/authentication.py

```diff
@@ -63,7 +63,34 @@ def decode_token(self):
                 'Invalid token. Please Provide a valid token!'
             }), 401
 
-    def save_user(self):
+    def andela_api_call(self, email, *expected_args):
+        try:
+            headers = {"Authorization": 'Bearer ' + self.get_token()}
+            if(os.getenv('APP_SETTINGS') == "testing"):
+                with open('users.json', 'r') as f:
+                    response = json.load(f)
+            else:  # pragma: no cover
```

Comment:
Is there a substantial reason why you are not covering this in your tests?

Reply:
This is a mock implementation of the Andela api being used for our tests.